ui/component/claimPreviewTile/view.jsx

```diff
@@ -141,10 +153,29 @@ function ClaimPreviewTile(props: Props) {
       className={classnames('card claim-preview--tile', {
         'claim-preview--channel': isChannel,
       })}
+      onMouseOver={handleOnMouseOver}
+      onMouseOut={handleOnMouseOut}
     >
-      <NavLink to={navigateUrl}>
-        <FileThumbnail thumbnail={thumbnailUrl} />
-      </NavLink>
+      <div>
+        <NavLink to={navigateUrl}>
+          <FileThumbnail thumbnail={thumbnailUrl} />
+        </NavLink>
+        <div
```

Comment:
Please use a css class for this. It can be added to `_card.scss` and called `card__absolute-actions`.

This also should only exist on the desktop app so this markup will need to be wrapped like this
```
{/* @if TARGET='app' */}
 <FileDownloadLink ... 
{/* @endif */}
```